src/libraries/System.Private.CoreLib/src/System/Threading/ThreadPoolWorkQueue.cs

```diff
@@ -404,12 +404,30 @@ public int Count
         private readonly int[] _assignedWorkItemQueueThreadCounts =
             s_assignableWorkItemQueueCount > 0 ? new int[s_assignableWorkItemQueueCount] : Array.Empty<int>();
 
+        private static object? _nextWorkItemToProcess;
```

Comment:
Regarding https://github.com/dotnet/runtime/pull/100506/commits/48c79129e791cc90bfa08f2c6bc337f5e4826998 and error CS0120, this field shouldn't need to be static, the singleton can be accessed via this static field: https://github.com/dotnet/runtime/blob/7c0f62fabce4e737e7b7d3c532dd4060b1d287a3/src/libraries/System.Private.CoreLib/src/System/Threading/ThreadPoolWorkQueue.cs#L1354